code/modules/mob/living/carbon/human/species_types/humans.dm

```diff
@@ -13,16 +13,31 @@
 	loreblurb = "Mostly hairless mammalians. Their home system, Sol, lies in a sort of \"bluespace dead-zone\" that blocks anything from entering or exiting Sol's dead-zone through bluespace without a relay. While it leaves Sol extremely well-defended, it meant that they went unnoticed and uncontacted until they were themselves able to breach it."
 
 /datum/species/human/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
+	if(C.dna.features["ears"] == "Elf")
+		mutantears = /obj/item/organ/ears/elf
 	if(C.dna.features["ears"] == "Cat")
 		mutantears = /obj/item/organ/ears/cat
+	if(C.dna.features["ears"] == "Dog")
+		mutantears = /obj/item/organ/ears/dog
 	if(C.dna.features["ears"] == "Fox")
 		mutantears = /obj/item/organ/ears/fox
+	if(C.dna.features["ears"] == "Rabbit")
+		mutantears = /obj/item/organ/ears/rabbit
+	if(C.dna.features["ears"] == "Bent Rabbit")
+		mutantears = /obj/item/organ/ears/rabbit/bent
+	if(C.dna.features["ears"] == "Floppy Rabbit")
+		mutantears = /obj/item/organ/ears/rabbit/floppy
 	if(C.dna.features["tail_human"] == "Cat")
 		mutant_organs |= /obj/item/organ/tail/cat
+	if(C.dna.features["tail_human"] == "Dog")
+		mutant_organs |= /obj/item/organ/tail/dog
 	if(C.dna.features["tail_human"] == "Fox")
 		mutant_organs |= /obj/item/organ/tail/fox
-	if(C.dna.features["ears"] == "Elf")
-		mutantears = /obj/item/organ/ears/elf
+	if(C.dna.features["tail_human"] == "Fox 2")
+		mutant_organs |= /obj/item/organ/tail/fox/alt
+	if(C.dna.features["tail_human"] == "Rabbit")
+		mutant_organs |= /obj/item/organ/tail/rabbit
```

Comment:
This is getting out of hand

Reply:
It is, but I'm not exactly sure of a better way here